### ISSUE TITLE
getExport will cause error with '--' in css

### ISSUE THREAD

**Issue by macisi** [egoist/rollup-plugin-postcss#27]
When my css is written like this
```css
.button--primary {
  composes: base;
  composes: primary;
}
.button--secondary {
  composes: base;
  composes: secondary;
}
.button--dashed {
  composes: base;
  composes: dashed;
}
```
The transformed code will cause error
``` js
export const button-Primary="_3hh9tei XY-M_dO _19BpMwo";
export const button-Secondary="vXJ9qrz XY-M_dO _3S37_fc";
export const button-Dashed="BNj9gmv XY-M_dO _35mc8WS";
```

**Comment by egoist** [comment]
Maybe we should only add `export default {}` by default and add a `namedExport` option to solve this for good?

/cc @lmihaidaniel @plrthink

**Comment by lmihaidaniel** [comment]
fixed by #34
